src/main/java/uk/gov/companieshouse/itemgroupworkflowapi/logging/LoggingUtilsConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.companieshouse.itemgroupworkflowapi.logging;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+@Configuration
+public class LoggingUtilsConfiguration {
+    public static final String APPLICATION_NAMESPACE = "item-group-workflow-api";
+    public static final String REQUEST_ID_HEADER_NAME = "X-Request-ID";
```

Comment:
Not sure a constant like `REQUEST_ID_HEADER_NAME` in particular belongs on anything to do with logging. I think it's used to access the contents of the header of this name?

In general, we might argue that even the other constants have little to do with the logging _configuration_ as such?